server/server.go

```diff
@@ -1016,6 +1016,11 @@ func (s *Server) unversionedGetHealth(w http.ResponseWriter, r *http.Request) {
 	includeBundleStatus := getBoolParam(r.URL, types.ParamBundleActivationV1, true) ||
 		getBoolParam(r.URL, types.ParamBundlesActivationV1, true)
 	includePluginStatus := getBoolParam(r.URL, types.ParamPluginsV1, true)
+	excludePlugin := getStringSliceParam(r.URL, types.ParamExcludePluginV1)
+	excludePluginMap := map[string]struct{}{}
+	for _, name := range excludePlugin {
+		excludePluginMap[name] = struct{}{}
+	}
```

Comment:
With this new param, the `health` request would look something like `/health?plugins&exclude-plugin=foo&exclude-plugin=bar`. An example in the docs would be great.